### ISSUE TITLE
visitable queryParam support

### ISSUE THREAD

**Issue by donovan-graham** [san650/ember-cli-page-object#39]
First off,  this is a great addon and a pleasure to work with.

I'd like to see `.visitable` being able to support `queryParams`. 
Query params are optional and would just be tacked onto the end of the existing url, if they are specified.

Here's the idea:

pages/books.js
```
export default PO.build({
  visit: visitable('/books/:category')
});
```

acceptance/books-test.js
```
import page from '../pages/books';

test('visiting adventure books', assert => {
  page.visit({category: 'adventure'},  {page: 3, limit: 10 });

  andThen(() => {
    assert.equal(currentURL(), '/books/adventure?page=3&limit=10');
  });
});
```

Possible fix
```
function visitable(params = {}, queryParams = {}) {

  let path = this.path;

  if (path.indexOf(':') !== -1) {
    path = fillInDynamicSegments(path, params);
  }

  if (Ember.keys(queryParams).length > 0) {
    path += "?" + Ember.$.param(queryParams);
  }

  visit(path);

  return this.page;
};
```

**Comment by donovan-graham** [comment]
Added pull request 
https://github.com/san650/ember-cli-page-object/pull/40

**Comment by san650** [comment]
Thanks Donovan, this is a great feature we're missing. I'll take a look at the PR.

**Comment by san650** [comment]
Closed by #40
